NEWS.md

```diff
@@ -135,6 +135,11 @@ For a more comprehensive changelog of the latest experimental code, see:
    - Fixed recenter the screen on activating an inventory item.
    - Fixed pressing W to talk to people also makes you jump.
 
+ Ultima8:
+   - Added support for saving and resizing of minimap.
+   - Adjust cursor to behave closer to the original.
+   - Adjust quantity slider to behave closer to the original.
```

Comment:
Can you provide more information regarding the functionality of the quantity slider?

Reply:
Would "item quantity slider" be better?

Reply:
> Would "item quantity slider" be better?

Yes, that sounds better :)